client/client-lib/src/wallet/mod.rs

```diff
@@ -340,10 +340,7 @@ mod tests {
         let output = PegOut {
             recipient: addr.clone(),
             amount,
-            fees: PegOutFees {
-                fee_rate: Feerate { sats_per_kvb: 1000 },
-                total_weight: 0,
-            },
+            fees: PegOutFees::new(1000, 883),
```

Comment:
This scares me a little since it would be very easy to reverse the order of these arguments ...